docs/content/querying/dimensionspecs.md

```diff
@@ -248,6 +248,57 @@ It is illegal to set `retainMissingValue = true` and also specify a `replaceMiss
 
 A property of `injective` specifies if optimizations can be used which assume there is no combining of multiple names into one. For example: If ABC123 is the only key that maps to SomeCompany, that can be optimized since it is a unique lookup. But if both ABC123 and DEF456 BOTH map to SomeCompany, then that is NOT a unique lookup. Setting this value to true and setting `retainMissingValue` to FALSE (the default) may cause undesired behavior.
 
+A property `optimize` can be supplied to allow optimization of lookup based extraction filter (by default `optimize = false`). 
+The optimization layer will run on the broker and it will rewrite the extraction filter as clause of selector filters.
+For instance the following filter 
```

Comment:
IMO if the reason it is off is that we want to use it in production first to build confidence, then it should be experimental, and maybe when it's not experimental anymore we should remove the setting and just have it always be on